newapi/ooniapi/auth.py

```diff
@@ -369,6 +367,25 @@ def user_login():
 """
 
 
+@metrics.timer("user_logout")
+@auth_blueprint.route("/api/v1/user_logout", methods=["GET"])
```

Comment:
I think we should make the logout method work only with `POST` requests. Otherwise it's possible to trigger a logout of any user by having them visit a page which loads something like:
```
<img src="https://api.ooni.io/api/v1/user_logout" />
```

The same is not possible if it's using POST, because that requires using `fetch` or `xhr`, which respects SOP.

Reply:
Updated